capture-sdk/sdk/src/main/java/net/gini/android/capture/review/multipage/MultiPageReviewActivity.java

```diff
@@ -45,9 +45,7 @@
  *
  * When you use the Screen API and have enabled the multi-page feature, the {@code
  * MultiPageReviewActivity} displays the photographed or imported images and allows the user to
- * review them by checking the order, sharpness, quality and orientation of the images. The user can
- * correct the order by dragging the thumbnails of the images and can also correct the orientation
- * by rotating the images.
+ * review them by checking the order, sharpness, quality and orientation of the images.
```

Comment:
[Request] You can replace this whole javadoc for MultiPageReviewActivity with `Internal use only`. It's not part of the public API anymore.

Sorry that I wasn't able to tell you that earlier. 🙈

Reply:
Done